debian/clickhouse-server.service

```diff
@@ -1,7 +1,8 @@
 [Unit]
 Description=ClickHouse Server (analytic DBMS for big data)
 Requires=network-online.target
-After=network-online.target
+After=time-sync.target network-online.target
```

Comment:
clickhouse-server should work even on servers with localhost only.

Reply:
Is it possible to place comments in this file?

Let's write comments around

```
[Install]
WantedBy=multi-user.target
```

because it is too cumbersome to read the documentation and learn systemd.

Reply:
>clickhouse-server should work even on servers with localhost only.

It will.

Reply:
>Is it possible to place comments in this file?

Sure.

>Let's write comments around

Ok.

Reply:
>because it is too cumbersome to read the documentation and learn systemd.

In a nutshell it is a basic of systemd, so adding a comment for `WantedBy=multi-user.target` will looks like the following in C:

```c
// initialize "a" to "1"
int a = 1;
```

But okay, I will do my best.